rest/data/commit_queue.go

```diff
@@ -298,60 +307,67 @@ func (pc *DBCommitQueueConnector) IsAuthorizedToPatchAndMerge(ctx context.Contex
 
 // EnqueuePRToCommitQueue enqueues an item to the commit queue to test and merge a PR.
 func EnqueuePRToCommitQueue(ctx context.Context, env evergreen.Environment, sc Connector, info commitqueue.EnqueuePRInfo) (*restModel.APIPatch, error) {
-	settings := env.Settings()
-	userRepo := UserRepoInfo{
-		Username: info.Username,
-		Owner:    info.Owner,
-		Repo:     info.Repo,
+	patchDoc, pr, err := getAndEnqueueCommitQueueItemForPR(ctx, env, sc, info)
+	if err != nil {
```

Comment:
`sendGitHubCommitQueueError` already does a nil check on the err. You could remove this and rename the function to something like `trySendGitHubCommitQueueError`

Reply:
Yeah it does check it, but I like writing sanity checks in places where I think a helper function might be used in the future in a way that's different from how I use it right now. It's nice to check inputs when you're not sure how someone later on is going to use it, especially since it would weird to send a failure status back to GitHub with no actual error message attached.